proto/core/store/store_object.proto

```diff
@@ -17,6 +17,10 @@ message StorableObject {
   google.protobuf.Any data = 4;
   string description = 5;
   repeated string tags = 6;
+  // fix by making Dict not StorableObject
```

Comment:
Q: Dict should be a storable object, right? :\ (because we can also use it as an object)

Reply:
StorableObject means description, tags, permissions though, surely we don't want that on every dict we send?